service/src/integrationTest/kotlin/fi/espoo/evaka/attendance/MobileUnitControllerIntegrationTest.kt

```diff
@@ -0,0 +1,136 @@
+// SPDX-FileCopyrightText: 2017-2022 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.attendance
+
+import com.github.kittinunf.fuel.jackson.responseObject
+import fi.espoo.evaka.FixtureBuilder
+import fi.espoo.evaka.FullApplicationTest
+import fi.espoo.evaka.insertGeneralTestFixtures
+import fi.espoo.evaka.placement.PlacementType
+import fi.espoo.evaka.shared.DaycareId
+import fi.espoo.evaka.shared.EmployeeId
+import fi.espoo.evaka.shared.GroupId
+import fi.espoo.evaka.shared.MobileDeviceId
+import fi.espoo.evaka.shared.PlacementId
+import fi.espoo.evaka.shared.auth.AuthenticatedUser
+import fi.espoo.evaka.shared.auth.UserRole
+import fi.espoo.evaka.shared.auth.asUser
+import fi.espoo.evaka.shared.dev.DevDaycareGroup
+import fi.espoo.evaka.shared.dev.createMobileDeviceToUnit
+import fi.espoo.evaka.shared.dev.insertTestDaycareGroup
+import fi.espoo.evaka.shared.dev.insertTestDaycareGroupPlacement
+import fi.espoo.evaka.shared.dev.insertTestPlacement
+import fi.espoo.evaka.shared.domain.HelsinkiDateTime
+import fi.espoo.evaka.testChild_1
+import fi.espoo.evaka.testChild_2
+import fi.espoo.evaka.testChild_3
+import fi.espoo.evaka.testChild_4
+import fi.espoo.evaka.testChild_5
+import fi.espoo.evaka.testDaycare
+import fi.espoo.evaka.withMockedTime
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.math.BigDecimal
+import java.time.LocalDate
+import java.time.LocalTime
+import java.util.UUID
+import kotlin.test.assertEquals
+
+class MobileUnitControllerIntegrationTest : FullApplicationTest(resetDbBeforeEach = true) {
+    private val mobileUser = AuthenticatedUser.MobileDevice(MobileDeviceId(UUID.randomUUID()))
+    private val groupId = GroupId(UUID.randomUUID())
+    private val groupName = "Testaajat"
+    private val today = LocalDate.of(2022, 2, 3)
+    private val now = HelsinkiDateTime.of(today, LocalTime.of(12, 5, 1))
+    private val placementStart = now.toLocalDate().minusDays(30)
+    private val placementEnd = now.toLocalDate().plusDays(30)
+    private lateinit var testEmployeeId_1: EmployeeId
+    private lateinit var testEmployeeId_2: EmployeeId
+
```

Comment:
None of these variables are used outside the data initialisation, so I don't think they need to be declared up here.